src/api.js

```diff
@@ -27,3 +27,15 @@ export function destroyIssue(id) {
     method: "delete",
   });
 }
+
+export function saveIssue(data) {
+  return fetch("/issues", {
+    method: "post",
+    body: JSON.stringify(data),
+    headers: {
+      "Content-Type": "application/json",
+    },
+  }).then((response) => {
+    return response.json();
+  });
+}
```

Comment:
In responses, a `Content-Type` header tells the client what the content type of the returned content actually is